src/app/scripts/cac/routing/cac-routing-itinerary.js

```diff
@@ -290,30 +292,41 @@ CAC.Routing.Itinerary = (function ($, cartodb, L, _, moment, Geocoder, Utils) {
     }
 
     /**
-     * Helper function to construct style object for an itinerary
+     * Helper function to set style for an itinerary
      *
      * @param {Boolean} shown Should this itinerary be shown (if false, make transparent)
      * @param {Boolean} highlighted Should this itinerary be highlighted on the map
-     * @return {Object} Leaflet style object to apply to geojson
      */
-    function getStyle(shown, highlighted) {
+    function setLineColors(shown, highlighted) {
+
         if (!shown) {
-            return {opacity: 0};
+            this.geojson.setStyle({opacity: 0});
+            return;
         }
+
         var defaultStyle = {clickable: true, // to get mouse events (listen to hover)
-                            color: '#d02d2d',
-                            dashArray: null,
-                            lineCap: 'round',
-                            lineJoin: 'round',
-                            opacity: 0.75};
+                        color: '#d02d2d',
```

Comment:
It would be nice to have the colors defined all in one place and not duplicated.  What about adding a `function getDefaultModeColor(highlighted) { ... }` to Utils?  Plus then you could just call that to set the color here and wouldn't have to replace it in the background case below.